actionview/lib/action_view/helpers/url_helper.rb

```diff
@@ -63,22 +63,23 @@ def _filtered_referrer # :nodoc:
       # of an options hash will generate a link to the referrer (a JavaScript back link
       # will be used in place of a referrer if none exists). If +nil+ is passed as the name
       # the value of the link itself will become the name.
+      # Use link_to with a block to wrap the whole block with your link.
       #
       # ==== Signatures
       #
-      #   link_to(body, url, html_options = {})
+      #   link_to(name, url, html_options = {})
```

Comment:
why change this from `body` to `name`? It's the link body that youa re setting, this can be arbitrary html. Also a tags do have a `name` attribute which makes this more confusing.